templates/team-base.html

```diff
@@ -39,21 +39,21 @@
 <form id="status-knob" action="set-status.json">
     <select name="status">
         <option {{ 'selected' if team.is_approved == None else '' }}
-            value="unreviewed">Under Review</option>
+            value="unreviewed">{{ _("Under Review") }}</option>
```

Comment:
To date, we haven't worried about translating our admin interface. Seems like scope creep to do so on this PR, no?